go-controller/pkg/ovn/policy.go

```diff
@@ -483,14 +483,14 @@ func (oc *Controller) createDefaultAllowMulticastPolicy() error {
 	err := addACLPortGroup("", oc.clusterRtrPortGroupUUID, fromLport,
 		defaultRoutedMcastAllowPriority, match, "allow", knet.PolicyTypeEgress, "")
 	if err != nil {
-		return fmt.Errorf("failed to create default deny multicast egress ACL: %v", err)
+		return fmt.Errorf("failed to create allow deny multicast egress ACL: %v", err)
```

Comment:
was this supposed to be "failed to create default allow multicast egress ACL" ?

Reply:
ah, yes, goofed fixing that. Let me push a fix

Reply:
fixed